src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/DisputantContactInformation.java

```diff
@@ -39,7 +39,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class DisputantContactInformation extends Auditable<String>{
```

Comment:
We might want to remove this class completely since it only had a reference in JJDispute model and now all the fields of this transferred to the JJDispute model.

Reply:
Yeah, I was thinking that as well.  A good cleanup task ... can do in a separate PR.